### ISSUE TITLE
'SeleniumScreenshot' object has no attribute 'css'

### ISSUE THREAD

**Issue by PaleNeutron** [dexplo/dataframe_image#89]
Hi, I tried using table_converter = 'selenium' and received the following error:

'SeleniumScreenshot' object has no attribute 'css'

The error is coming inside the def run(self, html) block. I had copy pasted the code in the _screenshot file that was posted by @wosantos95 for exporting table images using selenium. @waterbear1996 

Any help with this would be appreciated. Thanks.

_Originally posted by @ksaxena26 in https://github.com/dexplo/dataframe_image/issues/79#issuecomment-1523369991_

**Comment by PaleNeutron** [comment]
@ksaxena26 please give me a minimal test case for this issue and I'll deal with it.

**Comment by ksaxena26** [comment]
[test_data.xlsx](https://github.com/dexplo/dataframe_image/files/11343049/test_data.xlsx)
[screenshot py alt code.txt](https://github.com/dexplo/dataframe_image/files/11343117/screenshot.py.alt.code.txt)

```
def format_table(styler, dataframe, colmap, colmap_vmax):
    
    styler.format(formatter = lambda x: "{0:.2f}%".format(100*x), na_rep='', precision = 0)
    styler.format(formatter = lambda x: "{0:.2f}".format(x), subset = 'Disb Amount', na_rep='')
    
    styler.background_gradient(cmap = colmap, 
                               axis = None,
                               vmin = dataframe.iloc[:, 1:].min().min(),
                               vmax = colmap_vmax)
    
    styler.set_properties(subset = 'Disb Amount', **{'background-color': 'white',
                                                     'color': 'black'}, overwrite=False) 
    
    styler.set_table_styles([{"selector": "th", "props": "line-height: 8px;"},
                             {"selector": "td", "props": "line-height: 8px; padding: 0.5"},
                             {'selector': 'td', 'props': 'text-align: center; font-size: 11px;'},
                             {'selector': 'th', 'props': 'font-size: 11px;'},
                             {'selector': 'th', 'props': 'column-width: 50px'},
                             {"selector": "th", 'props': "background-color: white; text-align: center"},
                            ], overwrite = False)
    
    styler.set_table_styles({'Disb Amount':[{'selector':'td', 'props':'font-weight: bold;'}]
                            }, overwrite=False)
    
    styler.highlight_null(null_color='white')
    
    return styler

test_data = pd.read_excel('test_data.xlsx', index_col=0)
test_data.index = test_data.index.strftime("%b'%y")
styler_obj = test_data.style.pipe(format_table, 
                                  test_data, 
                                  'RdYlGn_r', 
                                  0.27)

dfi.export(styler_obj, 'styled_table.png', table_conversion="selenium", max_rows=-1)
```

The error comes at the last line in dfi.export. I have also attached the screenshot.py file that I am using.

**Comment by PaleNeutron** [comment]
This is a problem with your modified code.

If you move to official version which already support selenium, it works well.

![styled_table](https://user-images.githubusercontent.com/8296331/235038433-23e78648-ca7f-4dc0-94dc-12085b09bcf1.png)

BTW, I suggest latest matplotlib converter released in v0.1.11, 

`dfi.export(styler_obj, 'styled_table_mpl.png', table_conversion="matplotlib")`

![styled_table_mpl](https://user-images.githubusercontent.com/8296331/235038930-6a1f8bee-afcf-48ba-b9c7-d0fb9c728d10.png)

**Comment by PaleNeutron** [comment]
This warning is all right, matplotlib is a re-implemented and simulate way of html to image and very limited css properties are supported.

But it works very well in most case.
